README.md

```diff
@@ -46,162 +46,313 @@ pip install -r requirements.txt
 
 #### Prerequisites
 
 - OpenSSL installed on your machine (in WSL).
+- An Azure Communication Service connection string
+- An Azure MailFrom email address connected to the Azure Communication Service resource
 
 #### Steps
 
 1. Navigate to the project's root directory.
-2. Run the following script:
+2. Run the initialization script:
 
    ```bash
-   ./generate_keys.sh
+   ./init_project.sh <your-test-app-id>
    ```
 
-   This will generate four `.pem` files in a folder called `keys`:
+   This script will:
 
-   - `server_private_key.pem`
-   - `server_public_key.pem`
-   - `client_private_key.pem`
-   - `client_public_key.pem`
+   - Generate keys in a `keys` folder for both the server and the specified app id.
+   - Copy `.env.template` to `.env`.
+   - Copy `.env.tests.template` to `.env.tests`.
 
 #### Note
 
-- If the keys already exist, the script will overwrite them.
+- If the keys or environment files already exist, the script will overwrite them.
+- Logs are written to `init.log`.
 
-### Configure environment variables
+### 2. Configure Environment Variables
 
-1. Create a .env file in the project's root directory.
+1. Open the `.env` file generated in the project's root directory.
 2. Generate a secret key:
 
-```bash
-openssl rand -hex 32
-```
-
-3. Add the following variables:
-
-```
-ALLOWED_EMAIL_DOMAINS=gc.ca
-SECRET_KEY=<your_secret_key_here>
-JWT_ACCESS_TOKEN_EXPIRES_MINUTES=2
-SESSION_TYPE=filesystem
-CLIENT_PUBLIC_KEYS_DIRECTORY=keys/
-SERVER_PRIVATE_KEY=keys/server_private_key.pem
-SERVER_PUBLIC_KEY=keys/server_public_key.pem
-SESSION_LIFETIME_MINUTES=4
-REDIRECT_URL_TO_LOUIS_FRONTEND=http://localhost:3000
-```
+   ```bash
+   openssl rand -hex 32
+   ```
 
-4. Replace `<your_secret_key_here>` with the generated key from step 2.
+3. Populate the following variables in the `.env` file. Example for tests and dev:
+
+   ```env
+   # Server settings
+   MEMBRANE_CORS_ALLOWED_ORIGINS=http://localhost:3000
+   MEMBRANE_LOGGING_LEVEL=DEBUG
+   MEMBRANE_LOGGING_FORMAT=[%(asctime)s] [%(levelname)s] [%(filename)s:%(lineno)d:%(funcName)s] - %(message)s
+   MEMBRANE_HEALTH_MESSAGE=Ok
+
+   # Frontend
+   MEMBRANE_FRONTEND=http://localhost:3000
+
+   # Secrets and Keys
+   MEMBRANE_SECRET_KEY=#your secret key generated at step 2
+   MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY=keys/
+   MEMBRANE_SERVER_PRIVATE_KEY=keys/server_private_key.pem
+   MEMBRANE_SERVER_PUBLIC_KEY=keys/server_public_key.pem
+   MEMBRANE_TEST_APP_PUBLIC_KEY=keys/testapp1_public_key.pem
+   MEMBRANE_TEST_APP_PRIVATE_KEY=keys/testapp1_private_key.pem
+
+   # JWT and Session settings
+   MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS=300
+   MEMBRANE_JWT_EXPIRE_SECONDS=300
+   MEMBRANE_SESSION_LIFETIME_SECONDS=300
+   MEMBRANE_SESSION_COOKIE_SECURE=true
+   MEMBRANE_SESSION_TYPE=null
+   MEMBRANE_TOKEN_BLACKLIST=
+   MEMBRANE_APP_ID_FIELD=app_id
+   MEMBRANE_DATA_FIELD=data
+   MEMBRANE_REDIRECT_URL_FIELD=redirect_url
+   MEMBRANE_EXPIRATION_FIELD=exp
```

Comment:
this is part of the standard so doesn't need to be configurable:

https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4

Reply:
done